.github/workflows/publish-canary.yml

```diff
@@ -0,0 +1,19 @@
+name: Release
+
+on:
+  pull_request:
+    branches:
+      - master
+      - dev
+
+concurrency:
+  # New commit on branch cancels running workflows of the same branch
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish:
+    name: Publish canary version
+    uses: ./.github/workflows/publish-common.yml
+    secrets: inherit
```

Comment:
[Note]

После того как этот PR будет влит в `dev`, в этом workflow можно будет добавить 

```yml
    uses: ./.github/workflows/publish-common.yml
    with:
      cmd: "bootstrap:since"
```